### PR TITLE
Add support contact details

### PR DESCRIPTION
**Pull request by rl-dbt**
## What are you changing in this pull request and why?
Add support contact details related to partner and customer inquiries regarding the dbt Semantic Layer

### CODE REVIEW

File: website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md [hunk 1]
```diff
@@ -651,6 +651,14 @@ The API will be a viable integration point with the dbt Semantic Layer. It will
 
 ## Contact us
 
+### For dbt Semantic Layer Support
+For partner and customer support, please reach out via [support@dbtlabs.com](mailto:support@dbtlabs.com). Please ensure the subject line includes:
+
+- "Semantic Layer"
+- The name of the partner software
+- The account ID of the customer, if the partner is making the inquiry
```
Comment (amychen1776) [inline website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md]:
```suggestion
- The dbt cloud account ID of the customer, if the partner is making the inquiry
```

File: website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md [hunk 2]
```diff
@@ -651,6 +651,14 @@ The API will be a viable integration point with the dbt Semantic Layer. It will
 
 ## Contact us
 
+### For dbt Semantic Layer Support
+For partner and customer support, please reach out via [support@dbtlabs.com](mailto:support@dbtlabs.com). Please ensure the subject line includes:
```
Comment (amychen1776) [inline website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md]:
```suggestion
For partner and customer support, please reach out via [support@dbtlabs.com](mailto:support@dbtlabs.com). Please ensure the message includes:
```

**Review by amychen1776 — DISMISSED** [review]
:shipit:

File: website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md [hunk 3]
```diff
@@ -651,6 +651,14 @@ The API will be a viable integration point with the dbt Semantic Layer. It will
 
 ## Contact us
 
+### For dbt Semantic Layer Support
```
Comment (mirnawong1) [inline website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md]: sentence case per style guide

```suggestion
### For dbt Semantic Layer support
```

File: website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md [hunk 4]
```diff
@@ -651,6 +651,14 @@ The API will be a viable integration point with the dbt Semantic Layer. It will
 
 ## Contact us
 
+### For dbt Semantic Layer support
+For partner and customer support, please reach out via [support@dbtlabs.com](mailto:support@dbtlabs.com). Please ensure the message includes:
+
+- "Semantic Layer"
+- The name of the partner software
+- The dbt Cloud account ID of the customer, if the partner is making the inquiry
```
Comment (runleonarun) [inline website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md]:
```suggestion
- The dbt Cloud account ID of the customer, if you are a partner making the inquiry
```

**Comment by mirnawong1** [comment]
hey @rl-dbt - i've made some small tweaks and corrected the email since support@dbtlabs.com isn't the right one. approved and feel free to merge

**Review by mirnawong1 — APPROVED** [review]

**Pull request merged**